services/backend/app/api/endpoints/recipes.py

```diff
@@ -88,24 +94,82 @@ def get_recipe_by_id(
     recipe_id: UUID,
     db: Session = Depends(deps.get_db),
 ) -> Any:
-    pass
+    """
+    Get recipe by id
+    """
+    recipe = crud.recipe.get_by_id(db=db, recipe_id=recipe_id)
+    if recipe:
+        return recipe
+    else:
+        raise HTTPException(status_code=400, detail="Recipe not exists.")
```

Comment:
although, maybe better return 404 here, what do you think?

Reply:
You are right